tsconfig.json

```diff
@@ -55,6 +55,8 @@
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
   },
   "exclude": [
-    "karma.conf.js"
+    "karma.conf.js",
+    "./instrumented",
+    "coverage"
```

Comment:
If both `instrumented` and `coverage` are folders, please specify them with the same format.